src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/Services/RegisterWrapper.cs

```diff
@@ -18,24 +23,30 @@ public class RegisterWrapper : IRegister
     {
         private readonly HttpClient _client;
         private readonly IHttpContextAccessor _contextaccessor;
+        private readonly PlatformSettings _platformSettings;
+
+        private const string SubscriptionKeyHeaderName = "Ocp-Apim-Subscription-Key";
```

Comment:
this could be moved to platformsettings so it only has to be defined a single time for all the wrapper classes.

Reply:
@acn-sbuad  done